test/e2e/network-interface-create.e2e.ts

```diff
@@ -20,7 +20,7 @@ test('can create a NIC with a specified IP address', async ({ page }) => {
 
   // fill out the form
   await page.getByLabel('Name').fill('nic-1')
-  await page.getByRole('button', { name: 'VPC' }).click()
+  await page.getByRole('button', { name: 'VPC' }).first().click()
```

Comment:
Nitpick, but it's better to handle these cases by being more specific in the selector. 

```suggestion
  await page.getByLabel('VPC', { exact: true }).click()
```
Unfortunately `getByRole('button', { name: 'VPC', exact: true })` didn't work because that's not the exact name, and neither did `getByPlaceholder` (presumably because it's a button, not an input, so it's not really a placeholder).

<img width="1067" alt="image" src="https://github.com/oxidecomputer/console/assets/3612203/243e4130-436c-4d79-95d3-493cf78f5615">